sdk/lib/rtl/actctx.c

```diff
@@ -4665,6 +4667,21 @@ void actctx_init(void)
     {
         process_actctx = check_actctx(handle);
     }
+
+    ctx.dwFlags  = 0;
+    ctx.hModule  = NULL;
+    ctx.lpResourceName = NULL;
+    ctx.lpSource = buffer;
+    wcscpy( buffer, SharedUserData->NtSystemRoot );
+    wcscat( buffer, L"\\winsxs\\manifests\\x86_microsoft.windows.systemcompatible_6595b64144ccf1df_5.82.2600.2982_none_deadbeef.manifest");
```

Comment:
StringCch, or even `RTL_UNICODE_STRING_BUFFER` that will grow dynamically? (See `SdbpMatchExe` for an example)